app/services/oa_workflow_service.rb

```diff
@@ -33,5 +33,14 @@ def workflow
     rescue StandardError
       file.update_column(:downloaded, false)
     end
+
+    ActivityInsightOAFile.needs_version_check.each do |file|
+      file.version_checked = true
+      file.save!
```

Comment:
Here again, I think it might be better to do 
```ruby
file.update_column(:version_checked, true)
```
instead of
```ruby
file.version_checked = true
file.save!
```